packages/nextra/src/components/pre.tsx

```diff
@@ -33,36 +25,26 @@ export const Pre = ({
     }
   }, [])
 
-  useEffect(() => {
-    const codeEl = preRef.current?.querySelector('code')
-
-    if (codeEl?.textContent) {
-      setCodeString(codeEl.textContent)
-    }
-  }, [preRef])
```

Comment:
we don't need this anymore since this component was refactored by @shuding to take `getValue` fn and not `value` prop